tests/metrics/network/network-metrics.sh

```diff
@@ -19,8 +19,7 @@
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 # Description:
-#  Test inter (docker<->docker) network bandwidths and jitters
-# using iperf2
+#  Test inter (docker<->docker) bidirectional network bandwidth using iperf2
```

Comment:
@jcvenegas yes it is ok, we are removing the rests of the test keeping only the bidirectional one :)